tezos_interop/tezos_interop.re

```diff
@@ -131,6 +145,27 @@ module Key_hash = {
   type t =
     | Ed25519(Ed25519.Hash.t);
 
+  let name = "Signature.Public_key_hash";
+
+  let title = "A Ed25519, Secp256k1, or P256 public key hash";
```

Comment:
maybe include a comment indicating that this is copied from https://gitlab.com/tezos/tezos/-/blob/master/src/lib_crypto/signature.ml#L58 ?

Reply:
This is not from this PR, it was merged on #31